### ISSUE TITLE
Restructured text badly placed badge

### ISSUE THREAD

**Issue by gitter-badger** [gitterHQ/readme-badger#2]
Looks like the badge was put in the wrong place.
https://github.com/gitter-badger/scikit-bio/blob/gitter-badge/README.rst

**Comment by asyncanup** [comment]
In general, try to place the badge BELOW stuff like Travis Build Status.
Thoughts?

**Comment by trevorah** [comment]
Almost all repos I've seen have put there badges on one line:
![screen shot 2015-02-22 at 00 17 08](https://cloud.githubusercontent.com/assets/298742/6316670/47a22d98-ba28-11e4-9163-a49ba366b6e4.png)

While it might be neat to have different badge "categories" on different lines, it's overkill for 99% of readmes. Especially as readme-badger is meant for *any* badge, not just gitter badges.

**Comment by asyncanup** [comment]
In my case (asyncanup/apper), i keep the badges after a one line
description of the project, but badger put the gitter badge even before the
project description. Would have been fine if it had put it along with other
badges, below the description.

Maybe that would be a good way to place, find the other badges and place
alongside them?

**Comment by LaPingvino** [comment]
Agree with that, for me it placed it below the title, while my other badge is before the title...

**Comment by asyncanup** [comment]
I can submit a pull request if devs think it's a good idea

**Comment by trevorah** [comment]
Yeah, sounds cool. You could probably make the assumption that an image link on a page is a badge.

**Comment by asyncanup** [comment]
https://github.com/gitterHQ/readme-badger/pull/3

**Comment by gordonwoodhull** [comment]
I agree, it should find the other badges.  For my project, [it put it with the text](https://github.com/dc-js/dc.js/pull/891) when all our badges are above the title.

**Comment by asyncanup** [comment]
In progress, about to push an update that recognizes badges across all parsers

**Comment by DarkGigaByte** [comment]
+1
